DOC-1737: RC: Connection Wizard (#2211)
RedisLabs/redislabs-docs: content/rc/rc-quickstart.md

At this point, you're viewing the **Configuration** details for your new database.  

To connect to your database, you need your username and password. For the default user, the username is `default`.

The **Security** section contains your **Default user password**.  By default, this is masked.  Select the eye icon to show or hide the password.    

{{<image filename="images/rc/database-fixed-configuration-security.png" width="75%" alt="The Security section of the Configuration tab of the database details page." >}}{{< /image >}}

> Once you have the username and password, select the **Connect** button to open the connection wizard.

Reviewer: ```suggestion
Once you have the username and password, select **Connect** to open the connection wizard.
```

![Connect button](/images/rc/connection-wizard-button.png#no-click "Connect button.")

> The connection wizard shows how to connect to your database with the following methods:

Reviewer: ```suggestion
The connection wizard provides the following database connection methods:
```

> - Using the `redis-cli` utility

Reviewer: ```suggestion
- redis-cli` utility
```

> - Using a [Redis client](https://redis.io/clients) for your preferred programming language

Reviewer: ```suggestion
- [Redis client](https://redis.io/clients) for your preferred programming language
```

> - Using [RedisInsight](https://redis.com/redis-enterprise/redis-insight/)

Reviewer: ```suggestion
- [RedisInsight](https://redis.com/redis-enterprise/redis-insight/)
```

    "world"
    ```

    To try other Redis commands, see the [commands reference](https://redis.io/commands/) for help.

### Redis client{#using-redis-client}

Different programming languages use different connection clients to interact with Redis databases, and each client has its own syntax and installation process. For help with a specific client, see the client's documentation.

The connection wizard provides code snippets to connect to your database with the following programming languages:

- node.js using [ioredis](https://github.com/luin/ioredis#quick-start)
- .NET using [StackExchange.Redis](https://stackexchange.github.io/StackExchange.Redis/)
- Python using [redis-py](https://github.com/redis/redis-py#redis-py)
- Java using [Jedis](https://github.com/redis/jedis#jedis)

See the [client list](https://redis.io/docs/clients/) to view all Redis clients by language.

#### Code example (Python)

> Here's how to connect to your database using the [redis-py](https://github.com/redis/redis-py#redis-py) library for Python.

Reviewer: ```suggestion
To connect to your database using the [redis-py](https://github.com/redis/redis-py#redis-py) library for Python:
```

1.  Install the Redis client if it is not already installed.

    ```sh
    $ sudo pip install redis
    ```

2.  In the connection wizard, under **Redis Client**, select Python from the **Select your client** menu.

3. Select **Copy** to copy the connection code for your database.

> 4. Add the copied code to your program and replace `<username>` and `<password>` with your username and password.

Reviewer: ```suggestion
4. Paste the code to your program and replace `<username>` and `<password>` with your username and password.
```

    ```python
    import redis

    r = redis.Redis(
      host='<host>',
      port=<port>,
      password='<password>')

    # Redis commands
    r.set('hello', 'world')
    print(r.get('hello'))
    ```

5. Run the program.

    ```sh
    $ python example_redis.py
    world
    ```

### RedisInsight{#using-redisinsight}

RedisInsight is a free Redis GUI that is available for MacOS, Windows, and Linux.

1. In the connection wizard, under **Redis Client**, select your operating system from the **Download RedisInsight** menu.

1. Select **Download** to download RedisInsight.

1. [Install RedisInsight]({{< relref "/ri/installing/install-redis-desktop" >}}).

> 1. After you install and open RedisInsight, select **Add Redis Database**.

Reviewer: ```suggestion
1. Open RedisInsight and select **Add Redis Database**.
```

1. In the connection wizard, under **RedisInsight Desktop**, select **Copy** to copy the connection information.

> 1. In RedisInsight, enter the copied connection information into the **Host** field. RedisInsight automatically populates the rest of the fields needed to connect to the database as the default user.

Reviewer: ```suggestion
1. In RedisInsight, paste the connection information into the **Host** field. 
  RedisInsight automatically populates the rest of the fields needed to connect to the database as the default user.
```
Add the second sentence as an indented line below the step.
